packages/docusaurus-plugin-client-redirects/package.json

```diff
@@ -32,6 +31,7 @@
     "@docusaurus/types": "2.0.0-beta.9"
   },
   "peerDependencies": {
+    "@docusaurus/core": "2.0.0-beta.9",
```

Comment:
This one is slightly unrelated.

As Docusaurus becomes more stable (especially as the RC release approaches), we should consider the possibility of avoiding declaring hard dep on the core in every plugin.

Reply:
Yes, I don't know if presets should have plugins as peer deps—it works both ways—but core should certainly operate separately from the plugins. We have the core explicitly listed in the deps of init template anyways.